Add button text defaults and ability to override

jdhines opened the pull request:
I added handlebars expressions for the button values and created defaults in the component.

Commit:
Add button text defaults and ability to override

wecc commented:
Nice addition! Could we name the properties `closeText: 'Close'` and `okText: 'OK'` (to match the action names)?

Would also be great if this could be used in `logout-modal.hbs`, something like this:

```
{{#my-modal title='Logout' ok='logout' okText='Logout' close='removeModal' closeText='Stay logged in'}}
  Are you sure you want to logout?
{{/my-modal}}
```

Commit:
Add button text config to logout modal

jdhines commented:
I changed the property names and set the defaults in the logout-modal template. I also removed the only test under tests/unit/views/ since there are no views and the test was failing (which isn't a big deal I realize with this repo).

wecc commented:
Thanks! Sorry for being so slow :(
